dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -539,6 +539,11 @@ $small-footer-standard-width: 400px;
   color: black;
 }
 
+.user_options .pairing_summary {
+  font-size: 0.8em;
+  line-height: 1.6em;
```

Comment:
I don't think so — `em` is relative to the inherited font size so it should adjust proportionally if the menu font size changes.